metaflow/decorators.py

```diff
@@ -208,7 +208,7 @@ def step_init(self, flow, graph, step_name, decorators, environment, datastore,
         """
         pass
 
-    def package_init(self, flow, step_name, environment):
+    def package_init(self, flow, step_name, environment, logger):
```

Comment:
Not sure this change is needed here. You can use the logger from step_init IIRC.